app/src/main/res/layout/all_episodes_fragment.xml

```diff
@@ -17,6 +17,41 @@
         tools:itemCount="13"
         tools:listitem="@layout/new_episodes_listitem" />
 
+    <LinearLayout
```

Comment:
Why do those fragments need a copy of the empty view while all others can include the xml file? I don't like code duplication ;) If we want to change the style of the empty views, we have to change multiple locations.

Reply:
@ByteHamster Do you want ```EmptyViewHandler``` class to handle empty views in episode fragments?

Reply:
If that is possible, it would be great if it could be used everywhere.

Reply:
So I did some research and I found ```RecyclerView``` doesn't support ```setEmptyView``` method due to which we won't be able to add the empty view as we did for ListView in ```EmptyViewHandler``` class. Since ListView supports ```setEmptyView```  we were easily able to add empty view (layout from xml) to it

Reply:
I think you can use `RecyclerView.getAdapter().registerAdapterDataObserver()`. Then add another `attachTo` method that can attach to `RecyclerView`. If you inflate the view in the `EmptyViewHandler` constructor, the `setMessage` function can directly operate on the `TextView` and can therefore be called even after attach was already called.

Reply:
I tried the way you recommened but it didn't worked for me. Can you try it?